assets/app/view/game/emergency_money.rb

```diff
@@ -24,7 +24,7 @@ def render_emergency_money_raising(player)
           children << h(:div, props, corp)
         end
 
-        children << render_bankruptcy
+        children << render_bankruptcy if @game.round.steps.find { |s| s.respond_to?(:process_bankrupt) }
```

Comment:
this doesn't look good

i think maybe your intention here is to 

@game.round.actions_for(entity).include?('bankrupt')

Reply:
Ok, modified the code.